src/Core/src/Platform/Android/MauiApplication.cs

```diff
@@ -24,6 +24,7 @@ public override void OnCreate()
 			RegisterActivityLifecycleCallbacks(new ActivityLifecycleCallbacks());
 
 			var mauiApp = CreateMauiApp();
+			mauiApp.StartBackgroundServices();
```

Comment:
```suggestion
			mauiApp.StartAsync().GetAwaiter().GetResullt();
```

I think blocking to start hosted services while initializing an entire application is reasonable if there's not good way to make app creation async. Like I mentioned in the issue, even with an async Program.Main(), the compiler synthesizes the real entry point which blocks.

`async void` should almost always be avoided because it hides errors and makes it difficult to observe when an operation has completed. What if a hosted service needs to do some initialization in `StartAsync()` before making the app ready to use?

Reply:
@halter73  Fixed!